### ISSUE TITLE
Floor function symbols ⌊ and ⌋ don't render properly in the pdf version of the spec

### ISSUE THREAD

**Issue by David-DiGioia** [KhronosGroup/Vulkan-Docs#1400]
In **12.3.2. Image Miplevel Sizing** the floor function symbols ⌊ and ⌋ don't render properly. They appear as the stand-in boxes that appear when a symbol is not supported by a font.

This is the pdf I'm looking at: https://www.khronos.org/registry/vulkan/specs/1.2-khr-extensions/pdf/vkspec.pdf

![image](https://user-images.githubusercontent.com/43307034/99854277-84304280-2b52-11eb-9b3e-d5b58f37dccd.png)

**Comment by oddhack** [comment]
Thanks, I'll take a look at it. Perhaps a font that's missing these symbols.

**Comment by oddhack** [comment]
Yes, turns out that asciidoctor-pdf has limited Unicode support in its available fonts. I am surprised we overlooked this until now. I have asked if they can enhance the fonts in https://github.com/asciidoctor/asciidoctor-pdf/issues/1832 but failing that we will have to look at arcane font-fallback strategies (or use text markup instead of special symbols).

**Comment by oddhack** [comment]
This should be fixed in the 1.2.167 spec update. I did have to roll a custom fallback font but hopefully asciidoctor-pdf will eventually add the missing glyphs back into their version.

**Comment by ahcox** [comment]
This is back. Present in 1.4.310. 
Found in Firefox internal PDF viewer and in Evince 42.3 on Ubuntu 22.04.5 x86_64.
In Firefox the symbols render as whitespace.
In Evince they render as boxes.
